### PR TITLE
add doc for zha fans

### PR DESCRIPTION
**Pull request by igorbernstein2**
**Description:**
Adds docs for the new zha fan platform. The platform doesn't have any configuration of its own, so the doc is pretty sparse (basically copied from zha lights and replaces lights/fans)

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/12289

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/fan.zha.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+description: "Instructions how to setup ZigBee Home Automation fans within Home Assistant."
+date: 2018-02-10 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: zigbee.png
+ha_category: Fan
+ha_iot_class: "Local Polling"
+---
```
Comment (frenck) [inline source/_components/fan.zha.markdown]: `ha_release` missing?

**Review by frenck — CHANGES_REQUESTED** [review]
@igorbernstein2 Looks good but left a single commit for you to look at 👍

File: source/_components/fan.zha.markdown (hunk 1 above)
Comment (igorbernstein2) [reply source/_components/fan.zha.markdown]: Not sure what version I should put here...the platform hasn't been merged yet and I'm not sure if the next release will be a minor or a patch release

Comment (frenck) [reply source/_components/fan.zha.markdown]: @igorbernstein2 This a new platform/component, those are never released in a minor release.
So please put in the upcoming next major release (`0.66`).

Comment (igorbernstein2) [reply source/_components/fan.zha.markdown]: done

**Review by balloob — APPROVED** [review]

**Pull request merged**